src/test/java/org/eolang/speco/SpecoTest.java

```diff
@@ -0,0 +1,165 @@
+                )
+            );
+        }
+    }
+
+    @Disabled
+    @Tag("fast")
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/speco/packs", glob = "**.yaml")
+    public void convertsFromEo(
+        final String pack,
+        @TempDir(cleanup = CleanupMode.NEVER) final Path temp) throws IOException {
+        final Map<String, Object> script = new Yaml().load(pack);
+        MatcherAssert.assertThat(
+            "Unexpected transformation result",
+            Files.readString(SpecoTest.runSpeco(script, temp).resolve("app.eo")),
+            Matchers.equalTo(
+                script.get("after").toString()
+            )
+        );
+    }
+
+    @Tag("slow")
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/speco/packs", glob = "**.yaml")
+    public void compilesFromEo(
+        final String pack,
+        @TempDir(cleanup = CleanupMode.NEVER) final Path temp) throws IOException {
+        final Map<String, Object> script = new Yaml().load(pack);
+        MatcherAssert.assertThat(
+            "Unexpected execution result",
+            SpecoTest.exec(SpecoTest.runSpeco(script, temp).toString()),
+            Matchers.equalTo(
+                script.get("result").toString().split("\\r?\\n")
+            )
+        );
+    }
+
+    /**
+     * Runs Speco.
+     *
+     * @param script Yaml data object
+     * @param temp Path to the temporary dir
+     * @return Path to the output dir
+     * @throws IOException Iff IO error
+     */
+    private static Path runSpeco(final Map<String, Object> script, final Path temp)
+        throws IOException {
+        final Path input = temp.resolve("input");
+        final Path output = temp.resolve("output");
+        Files.createDirectories(input);
+        Files.write(
+            input.resolve("app.eo"),
+            script.get("before").toString().getBytes(),
+            StandardOpenOption.CREATE
+        );
+        new Speco(input, output, true).exec();
+        return output;
+    }
+
+    /**
+    * Compiles EO program.
+    *
+    * @param target Path to the dir with target EO program
+    * @return List of lines in output
+    * @throws IOException Iff IO error
+    */
+    private static String[] exec(final String target) throws IOException {
+        final String executor;
+        final String flag;
+        if (SystemUtils.IS_OS_WINDOWS) {
+            executor = "cmd";
+            flag = "/c";
+        } else {
+            executor = "bash";
+            flag = "-c";
+        }
+        final Process process = new ProcessBuilder(
+            executor,
+            flag,
+            String.format("eoc link -s %s && eoc --alone dataize app && eoc clean", target)
+        ).start();
+        final StringWriter writer = new StringWriter();
+        IOUtils.copy(process.getInputStream(), writer);
+        final String[] output = writer.toString().split("\\r?\\n");
+        return Arrays.copyOfRange(output, 11, output.length - 1);
```

Comment:
@advasileva `11` is a magic number. It's better to create a static final field and add `JavaDoc` for it with explanation "why it is exactly 11".